code/gradle.properties

```diff
@@ -20,5 +20,5 @@ appName = no.bouvet.app.imagelightbox
 displayName = Image lightbox
 vendorName = Bouvet Norge AS
 vendorUrl = http://www.bouvet.no
-xpVersion = 7.9.0
-version = 2.0.4
+xpVersion = 7.14.0
+version = 2.0.1
```

Comment:
Hvorfor ruller vi tilbake til 2.0.1 fra 2.0.4?🤔

Reply:
En feil, det skal ikke endres, tror jeg bare gjorde det for å være sikker på at jeg fikk ny versjon når den ble testet.